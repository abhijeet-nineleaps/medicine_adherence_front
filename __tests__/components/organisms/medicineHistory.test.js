import Enzyme, { shallow } from "enzyme";
import Adapter from "@wojtekmaj/enzyme-adapter-react-17";
import React from "react";
import toJson from "enzyme-to-json";
import MedicinehistoryList from "../../../src/components/organisms/medicineHistoryList";

Enzyme.configure({ adapter: new Adapter() });

describe("test collector category", () => {
  it("test category", () => {
    const wrapper = shallow(<MedicinehistoryList />).childAt(0).dive();
    expect.assertions(1);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
  it("test category", () => {
    let props = { item: { key: { not_taken: [{},{},{}] } }, showimgfun: "", medName: "" }
    const wrapper = shallow(<MedicinehistoryList {...props} />).childAt(0).dive();
    wrapper.find("#touchable1").props().onPress();
  });
  it("test category", () => {
    let props = { item: { key: { taken: [{},{},{}] } }, showimgfun: "", medName: "" }
    const wrapper = shallow(<MedicinehistoryList {...props} />).childAt(0).dive();
    wrapper.find("#touchable1").props().onPress();
  });
});