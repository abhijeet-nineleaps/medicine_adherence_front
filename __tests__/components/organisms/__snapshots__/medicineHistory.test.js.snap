// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test collector category test category 1`] = `
<ForwardRef(AnimatedComponentWrapper)
  style={
    Array [
      undefined,
      Object {
        "opacity": 0,
        "transform": Array [
          Object {
            "scale": 0.1,
          },
          Object {
            "translateY": 1000,
          },
        ],
      },
      Object {},
    ]
  }
>
  <View
    key="undefined1"
    style={
      Object {
        "marginBottom": 15,
        "padding": 4,
      }
    }
  >
    <withTheme(Card)
      key="undefined2"
      style={
        Object {
          "borderRadius": 6,
          "elevation": 2,
          "marginTop": 8,
          "padding": 14,
        }
      }
    >
      <View
        style={
          Object {
            "flexDirection": "row",
            "justifyContent": "space-between",
          }
        }
      >
        <View
          key="undefined3"
          style={
            Object {
              "flexDirection": "row",
              "justifyContent": "space-around",
            }
          }
        >
          <Text
            key="undefined7"
          >
            Date - 
          </Text>
        </View>
        <View>
          <TouchableOpacity
            id="touchable1"
            onPress={[Function]}
          >
            <Icon
              allowFontScaling={false}
              name="image"
              size={20}
            />
          </TouchableOpacity>
        </View>
      </View>
    </withTheme(Card)>
  </View>
</ForwardRef(AnimatedComponentWrapper)>
`;
